functions/README.md

```diff
@@ -46,77 +44,81 @@ cloud functions.
 
 1. Install [Firebase CLI](https://firebase.google.com/docs/cli/) globally to your system:
 
-    **Note:** This step is required even if you've already installed the tools for use with the web/ component.
-  
-    ```
-    $ npm install -g firebase-tools
-    ```
-  
-    For more information on using `npm install`, see <https://docs.npmjs.com/cli/install>
-  
-    For more information on firebase-tools, see <https://github.com/firebase/firebase-tools>
+   **Note:** This step is required even if you've already installed the tools for use with the web/ component.
+
+   ```
+   $ npm install -g firebase-tools
+   ```
+
+   For more information on using `npm install`, see
+   <https://docs.npmjs.com/cli/install>
+
+   For more information on firebase-tools, see
+   <https://github.com/firebase/firebase-tools>
 
 2. Log into Firebase:
 
-    ```
-    $ firebase login
-    ```
-  
-    After you run the `login` command, follow the prompts in your browser  or terminal to authenticate your Firebase account.
+   ```
+   $ firebase login
+   ```
+
+   After you run the `login` command, follow the prompts in your browser or
+   terminal to authenticate your Firebase account.
 
 3. Select your Firebase project:
 
-    ```
-    $ firebase use --add <project-name>
-    ```
-  
-    Where `<project-name>` is the name of a project associated with your firebase
-    account.
+   ```
+   $ firebase use --add <project-name>
+   ```
+
+   Where `<project-name>` is the name of a project associated with your Firebase
```

Comment:
Perhaps it is worth mentioning that the developer can reuse one of the ground projects from a list or something?

Reply:
This note refers to Firebase projects, not Ground projects. Developers will need to set up a project before going through this process; we should probably document this somewhere. Files #507 to track.